features/lorawan/LoRaWANBase.h

```diff
@@ -52,15 +82,44 @@ class LoRaWANBase {
      */
     virtual lorawan_status_t connect() = 0;
 
-    /** Connect OTAA or ABP by parameters
-     *
-     * Connect by Over The Air Activation or Activation By Personalization.
-     * The connection type is selected using the parameters.
-     * You need to define the parameters in the main application.
-     *
-     * @param connect       Options how end-device will connect to gateway
-     *
-     * @return    For ABP:  If everything goes well, LORAWAN_STATUS_OK is returned for first call followed by
+    /** Connect OTAA or ABP with parameters
+     *
+     * All connection parameters are chosen by the user and provided in the
```

Comment:
Note: Sometimes we say "you," and sometimes we say "the user". If the reader is the user, it should always be "you". If the reader's not the user, who is the reader?